docs/serializations.md

```diff
@@ -81,11 +81,58 @@ more serialized forms of itself.
 
 When serializing a physical SpdxDocument, any property of the logical element
 that can be natively represented within the chosen serialization format
-(e.g., `@context` prefixes in JSON-LD instead of the namespaceMap) may utilize
+(e.g., `@context` prefixes in JSON-LD instead of the namespaceMap) MAY utilize
 these native mechanisms. All remaining properties shall be serialized within
 the SpdxDocument element itself.
 
-A serialization must not contain more than one SpdxDocument.
+A serialization MUST NOT contain more than one SpdxDocument.
 
-A given instance of serialization must not define more than one SpdxDocument
+A given instance of serialization MUST NOT define more than one SpdxDocument
 element.
+
+## Serialization in JSON-LD
+
+### JSON-LD context file
+
+JSON-LD contexts allow JSON documents to use simple, human-readable, locally
+defined terms while ensuring data interoperability across different systems.
+
+The SPDX global JSON-LD context file MUST be used universally for all SPDX
+documents in JSON-LD format that adhere to a specific SPDX version.
+
+SPDX global JSON-LD context file is available at:
+[https://spdx.org/rdf/3.0.1/spdx-context.jsonld](https://spdx.org/rdf/3.0.1/spdx-context.jsonld)
```

Comment:
Just a note: This link is currently not live - if I can get the generated schema and context files, I can update the site and make them available.